Buildings/Controls/OBC/ASHRAE/FanCoilUnit/Controller.mo

```diff
@@ -0,0 +1,1160 @@
+        Text(
+          extent={{-200,-28},{-152,-50}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="TSup"),
+        Text(
+          extent={{-198,-132},{-152,-104}},
+          textColor={244,125,35},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="nOcc"),
+        Text(
+          visible=have_winSen,
+          extent={{-198,-186},{-154,-208}},
+          textColor={255,0,255},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uWin"),
+        Text(
+          extent={{132,-22},{198,-56}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="TAirSup"),
+        Text(
+          extent={{150,174},{198,150}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="yFanSpe"),
+        Text(
+          extent={{116,136},{194,102}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="TZonHeaSet"),
+        Text(
+          extent={{116,98},{196,60}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="TZonCooSet"),
+        Text(
+          extent={{142,60},{196,22}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="yHeaCoi"),
+        Text(
+          extent={{142,20},{196,-18}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="yCooCoi"),
+        Text(
+          extent={{-194,298},{-120,266}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="warUpTim"),
+        Text(
+          extent={{-194,258},{-114,226}},
+          textColor={0,0,127},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="cooDowTim"),
+        Text(
+          extent={{-194,54},{-82,32}},
+          textColor={255,127,0},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uCooDemLimLev"),
+        Text(
+          extent={{-194,12},{-82,-10}},
+          textColor={255,127,0},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uHeaDemLimLev"),
+        Text(
+          extent={{-202,-150},{-156,-170}},
+          textColor={255,0,255},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="uFan"),
+        Text(
+          extent={{158,212},{202,190}},
+          textColor={255,0,255},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          textString="yFan"),
+        Text(
```

Comment:
add a `visible` attribute, as the connector would be disabled. The text should not be visible when the connector is disabled. Please also check other connectors that could be disabled. Their text should become invisible when disabled.

Reply:
This should also be done for the subsequences.